examples/src/main/kotlin/io/grpc/examples/animals/AnimalsClient.kt

```diff
@@ -20,33 +20,28 @@ import io.grpc.ManagedChannel
 import io.grpc.ManagedChannelBuilder
 import java.io.Closeable
 import java.util.concurrent.TimeUnit
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.asExecutor
-import kotlinx.coroutines.async
-import kotlinx.coroutines.coroutineScope
-import kotlinx.coroutines.runBlocking
 
-class AnimalsClient constructor(private val channel: ManagedChannel) : Closeable {
+class AnimalsClient(private val channel: ManagedChannel) : Closeable {
     private val dogStub: DogGrpcKt.DogCoroutineStub by lazy { DogGrpcKt.DogCoroutineStub(channel) }
     private val pigStub: PigGrpcKt.PigCoroutineStub by lazy { PigGrpcKt.PigCoroutineStub(channel) }
     private val sheepStub: SheepGrpcKt.SheepCoroutineStub by lazy { SheepGrpcKt.SheepCoroutineStub(channel) }
 
-    suspend fun bark() = coroutineScope {
+    suspend fun bark() {
         val request = BarkRequest.getDefaultInstance()
-        val response = async { dogStub.bark(request) }
```

Comment:
So the rationale here is the async/await pieces are contrived and not a good fit for this example?

Reply:
Since `dogStub.bark` is already a suspend fun, and inside a suspend fun, this doesn't need to be explicit.